packages/flutter/test/material/button_theme_test.dart

```diff
@@ -402,4 +405,86 @@ void main() {
   },
     semanticsEnabled: true,
   );
+
+  testWidgets('default button theme accent color for RaisedButton', (WidgetTester tester) async { 
+    const Color defaultColor = Color(0xff2196f3);
```

Comment:
```suggestion
    const Color defaultEnabledAccentTextColor = Color(0xff2196f3);
```

Reply:
changed the variable name in all the tests